src/components/LayoutComposer/LayoutSideNavigation/LayoutSideNavigation.js

```diff
@@ -63,7 +65,11 @@ const LayoutSideNavigation = props => {
             <Main as="div" className={containerClasses}>
               <aside className={classNames(css.sideNav, sideNavClassName)}>
                 {useAccountSettingsNav ? (
-                  <LayoutWrapperAccountSettingsSideNav currentPage={currentPage} />
+                  <LayoutWrapperAccountSettingsSideNav
+                    currentPage={currentPage}
+                    showPaymentMethods={showPaymentMethods}
+                    showPayoutDetails={showPayoutDetails}
+                  />
                 ) : null}
```

Comment:
After live discussion: Let's create **_accountSettingsNavProps_** wrapper that passes all these props through the in-between components.

(LayoutWrapperAccountSettingsSideNav has translation keys, which makes it quite significant change to move this to come entirely through **_sideNavContent_** prop)

Reply:
Updated to use **accountSettingsNavProps**!